text/0976-glint-as-default.md

```diff
@@ -0,0 +1,161 @@
+
+**Integration Work:** Collaborate with the Glint project (which is maintained by Ember’s TypeScript and Tooling Core Teams) to ensure smooth integration into Ember.js, addressing any compatibility issues that may arise.
+
+**Community Outreach:** Organize blog posts and conference talks to educate the Ember.js community about Glint and its advantages.
+
+## Drawbacks
+
+> Why should we not do this? Please consider the impact on teaching Ember,
+on the integration of this feature with other existing and planned features,
+on the impact of the API churn on existing apps, etc.
+>
+
+> There are tradeoffs to choosing any path, please attempt to identify them here.
+>
+
+There are some potential drawbacks to adopting Glint:
+
+**Learning Curve:** Developers who are not familiar with TypeScript may face a learning curve when integrating Glint into their Ember projects.
+
+**Migration Effort:** Existing projects may require effort to migrate from not having a template checking system to Glint.
+
+**********************************************Registries Boilerplate:**********************************************
+
+Because Ember's template resolution occurs dynamically at runtime when using `.hbs` files, Glint needs a way of mapping the names used in your templates to the actual backing value they'll be resolved to. This takes the form of a "type registry" as defined above.
+
+Note that this is *not* an issue for users using `.gts` templates.
+
+## Alternatives
+
+> What other designs have been considered? What is the impact of not doing this?
+>
+
+> This section could also include prior art, that is, how other frameworks in the same domain have solved this problem.
+>
+
+The primary alternative to adopting Glint is to continue using Ember without a template type-checking system. However, this would mean missing out on the benefits of static type checking and improved tooling provided by Glint.
+
+## Unresolved questions
+
+> Optional, but suggested for first drafts. What parts of the design are still
+TBD?
+>
+
+### Is there redundancy between Ember Language Server and Glint Language Server?
+
+We should verify that there are no redundancies between the Glint Language Server and the Ember Language Server used in IDEs. If there is, we should answer the following questions:
+
+Should Glint Language Server features be merged into the Ember Language Server? Alternatively, should we remove the redundancy between them but keep them separate?
```

Comment:
I do think it would be good to have only one language server to install.

Reply:
Is there any overlap? If not, there's still benefit in merging, but it does become a somewhat tangential concern.